package.json

```diff
@@ -35,6 +35,7 @@
     "@types/express": "^4.17.2",
     "@types/helmet": "^0.0.45",
     "@types/http-proxy-middleware": "^0.19.3",
+    "@types/node": "^13.7.0",
```

Comment:
We dont need this if we drop the stuff in the mock server right?

Reply:
ok